<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_64_128_5_train">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_64_128_5_test">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_64_128_5_show_weights">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Weights</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_64_128_5_random_initilize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Initilize</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_64_128_5_random_initilize">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-----------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_64_128_5_train">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-----------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_64_128_5_test">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-----------------</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_64_128_5_save_weights">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Weights</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_64_128_5_show_weights">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-----------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_64_128_5_save_weights">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-----------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_64_128_5_load_saved_weights">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-----------------</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_64_128_5_load_saved_weights">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Saved Weights</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
